test/httpwstest.cpp

```diff
@@ -149,11 +148,6 @@ void HTTPWSTest::testSaveOnDisconnect()
         sendTextFrame(socket1, "paste mimetype=text/plain;charset=utf-8\n" + text, testname);
         getResponseMessage(socket1, "pasteresult: success", testname);
 
-        std::set<pid_t> origPids = getDocKitPids();
-        LOK_ASSERT_EQUAL(static_cast<size_t>(1), origPids.size());
```

Comment:
Why did we loose this ?

Reply:
Verifying specific pids shouldn't be necessary. Perhaps in one specific test to verify that they don't get reused (is that even possible?) but no need to have lots of extra code to do that in unrelated tests.

It was here because it was the initial fix for the recent unit-httpws failure #8463. If that fix and this one were done together (and they will be backported together) there would have been no need to add it.

Reply:
The assert checks not so much the pid, but that there is still just one process open; that seems a useful check surely.

Reply:
The size check was only there to make `*(origPids.begin())` safe. Otherwise it would have gotten a random or null(?) pid.

Like checking that it doesn't reuse pids, checking that it doesn't open two kits could be useful, but that's not what this test is for. I made a note and I'll add those new testpoints later.